codetf.schema.json

```diff
@@ -0,0 +1,228 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://pixee.ai/codetf.schema.json",
+  "title": "Code Transformation Format (CodeTF)",
+  "description": "A format for representing the results of code transformations",
+  "type": "object",
+
+  "properties": {
+    "run": {
+      "description": "Metadata about the run that produced the results",
+      "$ref": "#/definitions/run"
+    },
+    "results": {
+      "description": "The results of the code transformations that were applied during analysis",
+      "type": "array",
+      "items": { "$ref": "#/definitions/result" },
+      "minItems": 0
+    }
+  },
+
+  "definitions": {
+    "run": {
+      "type": "object",
+      "properties": {
+        "vendor": {
+          "type": "string",
+          "description": "Name of the organization that produced the tool"
+        },
+        "tool": {
+          "type": "string",
+          "description": "Name of the tool that produced the CodeTF results"
+        },
+        "version": {
+          "type": "string",
+          "description": "Version of the tool that produced the CodeTF results"
+        },
+        "projectName": {
+          "type": "string",
+          "description": "Name of the project that was analyzed"
+        },
+        "commandLine": {
+          "type": "string",
+          "description": "The command line used to re-create this run"
+        },
+        "elapsed": {
+          "type": "string",
```

Comment:
number

Reply:
Same reply as above.

Reply:
```
  /Users/jgilday/Downloads/results.codetf-12.json::$.run.elapsed: 67311 is not of type 'string'
```

Reply:
I see it too. That's good news for us, although Python is generating this stuff as a string per the spec. Luckily it seems like the platform is not sensitive to that difference.

Reply:
I'm surprised the platform isn't failing on this.